flterm.c

```diff
@@ -206,6 +232,103 @@ static int upload_fd(int serialfd, const char *name, int firmwarefd, unsigned in
 	return length;
 }
 
+static int upload_xmodem(int serialfd, const char *name, int firmwarefd)
```

Comment:
Can you add a doxygen style comment about this function?

There is something about only supporting 1k packets or something?